modules/hydra-master-main.nix

```diff
@@ -79,13 +79,13 @@ in {
         input-output-hk = ${builtins.readFile ../static/github_token}
       </github_authorization>
       <githubstatus>
-        jobs = serokell:iohk-ops.*
+        jobs = serokell:iohk-ops.:required
         inputs = jobsets
         excludeBuildFromContext = 1
         useShortContext = 1
       </githubstatus>
       <githubstatus>
-        jobs = serokell:cardano.*
+        jobs = serokell:cardano.*:required
```

Comment:
the release branch of cardano will need a PR before this line is merged, or no status's will update